tests/vmi_lifecycle_test.go

```diff
@@ -986,8 +983,6 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			var supportedKVMInfoFeature []string
 
 			BeforeEach(func() {
-				// arm64 does not support cpu model
-				checks.SkipIfARM64(testsuite.Arch, "arm64 does not support cpu model")
 				nodes = libnode.GetAllSchedulableNodes(kubevirt.Client())
```

Comment:
TBH I am not completely familiar with these tests, maybe @Barakmor1 or @iholder101 can take another look?

Reply:
Seems like it is running only by periodic sig-compute lanes:
https://search.ci.kubevirt.io/?search=test_id%3A3202&maxAge=336h&context=1&type=all&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

Looking at `test.sh`, it seems that sig-compute doesn't skip the `CPUModel` decorator. I also don't see any skips on the period job definition [1].

Are there any other concerns?

[1] https://github.com/kubevirt/project-infra/blob/15f534cd90d00eba18562f93b9fd9d34a611418f/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L2267-L2287

Reply:
In libvirt, only the host-passthrough CPU model is supported on the Arm64 architecture. As a result, we have skipped all tests that attempt to set a specific CPUModel.